packages/mirai/lib/src/framework/mirai_registry.dart

```diff
@@ -16,24 +16,14 @@ class MiraiRegistry {
 
   bool register(MiraiParser parser) {
```

Comment:
@khanjasir90 Can we add an optional param called "bool override". And the parser will be overridden only of the value is true.

Reply:
Sure, added the same @divyanshub024